<?xml version="1.0" ?>
<launch>
    <arg name="robot_name" default="pioneer_0"/>

    <include file="$(find swarmus_ros_navigation)/launch/navigation.launch"/>
    <include file="$(find swarmus_ros_simulation)/launch/slam_simulation.launch"/>

    <node pkg="hiveboard_bridge" type="hiveboard_bridge" name="hiveboard_bridge" output="screen">
        <rosparam file="$(find hive_mind_bridge)/config/hive_mind_bridge_params.yaml" command="load"/>
        <param name="ROBOT_NAME" type="str" value="$(arg robot_name)"/>
    </node>

    <node name="hive_mind" pkg="hive_mind" type="hive_mind"  output="screen" >
        <param name="board_uuid" value="1" />

        <param name="host_tcp_address" value="127.0.0.1" />
        <param name="host_tcp_port" value="5555" />

        <!-- Set port to 0 to disable server -->
        <param name="uart_mock_port" value="12345" />

        <param name="log_level" value="Debug" />
    </node>

</launch>